src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -181,28 +194,28 @@ module MOM_ice_shelf
 
   logical :: debug                !< If true, write verbose checksums for debugging purposes
                                   !! and use reproducible sums
+  integer :: id_clock_shelf=-1 !< CPU Clock for the ice shelf code
```

Comment:
CPU clocks are typically treated as module variables, because they are not instance specific, but instead track the time in code, and should be summed across all ensemble members.  These CPU clocks are among the very few MOM6 prohibitions against the use of module variables.

Reply:
Thanks!